### ISSUE TITLE
The requested URL /thankslist was not found on this server

### ISSUE THREAD

**Issue by Galixte** [rxu/thanks_for_posts#18]
Hi,

on the index of my forum when i click on the link ( Thanks Toplist — %d) : 

```
http://mydomain.tld/thankslist 
```

I have this error :

```
Not Found

The requested URL /thankslist was not found on this server.
```

**Comment by rxu** [comment]
Fixed with 24319e8f4abe6939fd35bbcc159460c0214d8bfd, thanks.

**Comment by Galixte** [comment]
You're welcome
